src/jdk.jpackage/macosx/classes/jdk/jpackage/internal/MacBaseInstallerBundler.java

```diff
@@ -113,6 +114,27 @@ static String getInstallDir(
         return returnValue;
     }
 
+    // Returns display name of installation directory. Display name is used to
+    // show user installation location and for well known (default only) we will
+    // use "Applications" or "JavaVirtualMachines". For user provided installation
+    // directory we will use full path as display name.
+    static String getInstallDirDisplayName(
```

Comment:
Is this method ever called with `false` as the last parameter? If no, I'd remove it.

Reply:
No, it does not. I removed it.